### PR TITLE
.NET Standard 2.0 / v4.2.0 version of the library.

### PR DESCRIPTION
**Pull request by adrianhall**

### CODE REVIEW

**Review by mattleibow — APPROVED** [review]
Things are looking really good. Many of the changes are just modernized C# syntax features, so that is nice.

The CI system is also making sure the code actually runs find on devices, so that is very confidence inducing,

I see there is an implementation using the `WebAuthenticationBroker` for UWP. If code reduction was an option, there is also the same API in Xamarin.Essentials - basically you should be able to share the same code with iOS, Android and UWP.

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# TODO: The maintainer of this repo has not yet edited this file
```
[the remark this answers is not in the document]
Comment (adrianhall) [reply SUPPORT.md:1]: Oops - I still have it on my machine - will check it in before I merge.

File: e2etests/DeviceTests.Shared/DeviceTests.Shared.csproj [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<Project Sdk="MSBuild.Sdk.Extras/2.1.2">
+  <PropertyGroup>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">Xamarin.iOS10;MonoAndroid90;</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">Xamarin.iOS10;MonoAndroid90;</TargetFrameworks>
```
[the remark this answers is not in the document]
Comment (adrianhall) [reply e2etests/DeviceTests.Shared/DeviceTests.Shared.csproj:4]: There are no additional tests beyond .NET standard 2.0 for UWP.  Those are tested as part of the unit tests.

**Comment by pooran** [comment]
Thank you!!!

**Pull request merged**